Examples/BookingSystem.AspNetFramework/Stores/FacilityStore.cs

```diff
@@ -250,9 +247,27 @@ protected override void GetOrderItems(List<OrderItemContext<FacilityOpportunity>
                                                  x.OrderTable.ProposalStatus != ProposalStatus.SellerRejected &&
                                                  x.SlotId == slot.Id &&
                                                  x.OrderId != flowContext.OrderId.uuid)
-                                     }
+                                     },
+                                     Attendee = flowContext.Customer != null
+                                         ? new Person
+                                         {
+                                             GivenName = flowContext.Customer.GivenName,
+                                             FamilyName = flowContext.Customer.FamilyName,
+                                             Email = flowContext.Customer.Email,
+                                             Telephone = flowContext.Customer.Telephone
+                                         }
+                                         : null,
```

Comment:
Don't need this - it should be set by the test suite on the request, not on the response

```suggestion
```